src/aere_repl.hrl

```diff
@@ -7,6 +7,7 @@
         , call_value   := non_neg_integer()
         , print_format := print_format()
         , print_unit   := boolean()
+        , locked_opts  => [atom()]
```

Comment:
Why are you using an optional association here? Why not `locked_opts  := [atom()]`?

Reply:
Because it has a natural default which is an empty list